client/checkout/api/index.js

```diff
@@ -74,10 +70,7 @@ export default class WCPayAPI {
 			if ( isUPEEnabled ) {
 				let betas = [ 'card_country_event_beta_1' ];
 				if ( isStripeLinkEnabled ) {
-					betas = betas.concat( [
-						'link_autofill_modal_beta_1',
-						'link_beta_2',
-					] );
+					betas = betas.concat( [ 'link_autofill_modal_beta_1' ] );
```

Comment:
Interesting, did we clarify with Stripe the need of `link_autofill_modal_beta_1`?

Reply:
> Interesting, did we clarify with Stripe the need of `link_autofill_modal_beta_1`?

I tested without this meta, and some errors were thrown. I will re-test it, and double check Stripe.